packages/ses/src/commons.js

```diff
@@ -137,6 +137,9 @@ export const { prototype: generatorPrototype } = getPrototypeOf(
   // eslint-disable-next-line no-empty-function, func-names
   function* () {},
 );
+export const iteratorPrototype = getPrototypeOf(
+  getPrototypeOf(arrayPrototype.values()),
```

Comment:
For this lint error, you should feel free to just suppress it. `commons.js` is initialized early enough that for anything that happens during its initialization, this hazard is a non-issue.

```suggestion
  // eslint-disable-next-line @endo/no-polymorphic-call
  getPrototypeOf(arrayPrototype.values()),
```